src/assets/translations.data.json

```diff
@@ -22,6 +22,7 @@
     "entry.invite-others": "invite others",
     "entry.invite-others-nag": "invite others to join",
     "entry.enable-screen-sharing": "Share my desktop",
+    "entry.return-to-vr": "Return to VR",
```

Comment:
I'd probably change this to "Enter in VR" to match the copy on the usual landing page